torchvision/transforms/functional.py

```diff
@@ -82,6 +82,38 @@ def to_tensor(pic):
         return img
 
 
+def convert_image_dtype(
+    image: torch.Tensor, dtype: torch.dtype = torch.float
+) -> torch.Tensor:
+    def scale_factor(dtype: torch.dtype) -> float:
+        if dtype in (
+            torch.float32,
+            torch.float,
+            torch.float64,
+            torch.double,
+            torch.float16,
+            torch.half,
+        ):
```

Comment:
Can we use instead `dtype.is_floating_point`?

Reply:
I did not know that this exist. Thanks, this will be much cleaner.